### DISCUSSION TITLE
Network v15 OhSnap 👏

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#436] [Network + Implementations Announcements]
Community discussion forum for the upcoming[ Network v15 OhSnap upgrade](https://github.com/filecoin-project/community/discussions/74#discussioncomment-1922550)!

**Comment by Fatman13** [comment]
What are the network params for snapnet? Will 512m be supported?

Edit:

Looks like 512m is supported? https://github.com/filecoin-project/lotus/pull/7932

**Comment by BlocksOnAChain** [comment]
Also, sharing more details around the trusted setup for v15 upgrade, it should give more details about the announcement we made: https://github.com/filecoin-project/community/discussions/440 - Trusted setup ceremony for v15 (OhSnap)
